ArduCopter/Parameters.cpp

```diff
@@ -932,6 +932,14 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     AP_SUBGROUPINFO(oa, "OA_", 33, ParametersG2, AP_OAPathPlanner),
 #endif
 
+    // @Param: FS_OPTIONS
+    // @DisplayName: Failsafe options bitmask
+    // @Description: Bitmask of additional options for battery, radio, & GCS failsafes. 0 (default) disables all options.
+    // @Values: 0:Disabled, 1:Continue mission on radio FS only, 2:Continue mission on GCS FS only, 3:Continue mission in radio and GCS FS, 67:Continue mission in radio and GCS FS + continue in pilot operated modes
```

Comment:
I think it would be good if the FC_OPTIONS param values and bitmasks items always started with the type of FS affected by the option.  I.e. "Radio failsafe continue in Auto"

Reply:
I tried this but I had a hard time getting them to make sense in context. Left them as is for now.